repo/packages/S/spark/1/marathon.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "spark",
```

Comment:
we are looking for id, cpus, mem, instances to all have config.json properties so they can be modified at installation